src/bluesearch/database/topic.py

```diff
@@ -297,6 +299,66 @@ def get_topics_for_pmc_article(
     return journal_topics
 
 
+def extract_info_from_zipfile(path: pathlib.Path) -> tuple[str, str]:
```

Comment:
Makes sense. I just noticed that out naming is not consistent actually.

```
git checkout master
git grep 'def extract' -- src/bluesearch/database/

src/bluesearch/database/topic.py:def extract_pubmed_id_from_pmc_file(path: str | pathlib.Path) -> str | None:
src/bluesearch/database/topic.py:def extract_article_topics_for_pubmed_article(
src/bluesearch/database/topic.py:def extract_journal_topics_for_pubmed_article(
```
Sometimes we use "for" and sometimes we use "from". But maybe this can be handled in a separate PR

Reply:
I used the name `extract_article_topics_from_medrix_article `and the fix is here: df022eb916b124776191d45df8f54bd1117e1731